main/__tests__/IconButton.test.tsx

```diff
@@ -110,11 +104,10 @@ describe('[IconButton]', () => {
       expect(json).toBeTruthy();
     });
 
-    it('should render type==="primary"', () => {
+    it('should render color==="primary"', () => {
```

Comment:
~~should render color==="primary"~~ => renders primary color

would be more concise